src/deploy/extensions/deploymentSummary.ts

```diff
@@ -45,6 +45,14 @@ export function updatesSummary(
   return toUpdate.length ? `The following extension instances will be updated:\n${summary}\n` : "";
 }
 
+export function configuresSummary(toConfigure: planner.InstanceSpec[]) {
+  return toConfigure.length
+    ? `The following extension instances will be configured:\n${toConfigure
+        .map((s) => `\t${humanReadable(s)}`)
+        .join("\n")}\n`
```

Comment:
a bit message, pull the inlined part into a variable?

Reply:
Done, and also did the same for the other summary methods.